pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -67,6 +69,9 @@ type TokensControllerOptions struct {
 	// MaxRetries controls the maximum number of times a particular key is retried before giving up
 	// If zero, a default max is used
 	MaxRetries int
+
+	// This CA will be added in the secrets of service accounts
+	ServiceServingCA []byte
```

Comment:
is this the service-ca operator CA?

Reply:
As per the conversation on slack around https://bugzilla.redhat.com/show_bug.cgi?id=1813894, I'm under the impression that this commit needs to be retained at least through the rebase.